README.md

```diff
@@ -94,15 +94,15 @@ Alternative error:
 ! was unexpected at this time.
 npm ERR! code ELIFECYCLE
 npm ERR! errno 1
-npm ERR! @google/model-viewer@1.10.1 prepare: `if [ ! -L './shared-assets' ]; then ln -s ../shared-assets ./shared-assets; fi && ../shared-assets/scripts/fetch-khronos-gltf-samples.sh`
+npm ERR! @google/model-viewer@1.10.1 prepare: `if [ ! -L './shared-assets' ]; then ln -s ../shared-assets ./shared-assets; fi && ../shared-assets/scripts/fetch-khronos-gltf-models.sh && && ../shared-assets/scripts/fetch-khronos-gltf-assets.sh`
```

Comment:
Can we take this opportunity to switch from the old sample-models repo to the new sample-assets repo instead of using both? The models are the same anyway.

Reply:
Would you be okay if I switch from this custom git checkout method to the more standard git submodule method?

Reply:
Done: https://github.com/google/model-viewer/pull/4534